README.md

```diff
@@ -7,6 +7,7 @@ It resembles well known tools such as _supervisord_, _runit_, or _s6_, in that i
 easily manage non-system processes independently from the system services, but it was
 designed with unique features that help with more specific use cases.
 
+  - [Installation](#installation)
```

Comment:
What about we just add a paragraph to "Using Pebble"?

I suggest adding it to the very top of that section, as follows:

    To install the latest version of Pebble, run the following command (we don't currently
    ship binaries, so you must first [install Go](https://go.dev/doc/install)):

    ```
    go install github.com/canonical/pebble/cmd/pebble@latest
    ```